scripts/build/ci-build/Dockerfile

```diff
@@ -108,7 +108,7 @@ RUN rm dockerize-linux-amd64-v${DOCKERIZE_VERSION}.tar.gz
 # Use old Debian (LTS into 2024) in order to ensure binary compatibility with older glibc's.
 FROM debian:buster-20220822
 
-ENV GOVERSION=1.21.3 \
+ENV GOVERSION=1.21.5 \
```

Comment:
@kminehart is this `Dockerfile` still in use?

Reply:
yea

Reply:
Can I remove the whole `publish_ci_build_container_image_pipeline` and it's references then?

Like that https://github.com/grafana/grafana/pull/79329/commits/a7bbedecef90a1026f68256c624603b29697db25? Please double check 🙏

Reply:
@kminehart